javaScript/find.smallest.js

```diff
@@ -0,0 +1,21 @@
+function smallest(num) {
+    let move = [num, 0, 0];
+    num = num.toString();
+    let smallestNum = num;
+    for (let i = 0; i < num.length; i++) {
+        for (let j = 0; j <= num.length; j++) {
+            let tempNum = num.split('');
```

Comment:
instead of splitting here could you get the number by index with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charAt